src/main/java/world/bentobox/level/commands/IslandTop.java

```diff
@@ -17,7 +17,9 @@ public IslandTop(Level plugin, CompositeCommand parent) {
 
     @Override
     public boolean execute(User user, String label, List<String> list) {
-        plugin.getTopTen().getGUI(getWorld(), user, getPermissionPrefix());
+        if (user.isPlayer()) {
```

Comment:
No. You should rather use `setOnlyPlayer(true);` in the command `setup()` method.